python/cudf/cudf/tests/test_s3.py

```diff
@@ -131,6 +131,7 @@ def pdf_ext(scope="module"):
     df["Integer"] = np.array([i for i in range(size)])
     df["List"] = [[i] for i in range(size)]
     df["Struct"] = [{"a": i} for i in range(size)]
+    df["String"] = ["Alpha", "Beta", "Gamma", "Delta"] * (size // 4)
```

Comment:
you can ceil and truncate to make it work for any size:
```suggestion
    df["String"] = (["Alpha", "Beta", "Gamma", "Delta"] * (-(size // -4)))[:size]
```